frontend/packages/dev-console/src/components/gitops/details/CommitDetails.tsx

```diff
@@ -21,7 +21,7 @@ const CommitDetails: React.FC<CommitDetailsProps> = ({ imageName }) => {
     kind: 'ImageStreamImport',
     apiVersion: 'image.openshift.io/v1',
     metadata: {
-      name: 'gitops-app',
+      name: `gitops-app`,
```

Comment:
No need for string literals here.
```suggestion
      name: 'gitops-app',
```

Reply:
Yeah I was using `imageName` before and that is why used string literals.